EnglishFilter/sections/general_extensions.txt

```diff
@@ -234,6 +234,8 @@ kat.*,kickass.*,kickass2.*,kickasstorrents.*,kat2.*,kattracker.*,thekat.*,thekic
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/77663
+uploadev.org#%#//scriptlet("abort-on-property-write", "Object.assign")
```

Comment:
It looks a bit wide. This also works `uploadev.org#%#//scriptlet("abort-on-property-read", "openInNewTab")`.

Reply:
Thanks!